frontend/src/routes/(authenticated)/project/create/+page.svelte

```diff
@@ -80,7 +83,8 @@
   }
 
   onMount(() => { // we want to do this once after the user has been set
-    const urlValues = getSearchParamValues<Partial<Omit<CreateProjectInput, 'id'>>>();
+    requestingUser = data.requestingUser;
```

Comment:
why are we doing this on mount instead of the root of the component? SSR won't work this way.

Reply:
Yeah...because I only want it to happen once. If I make it reactive:
`$: requestingUser = data.requestingUser;`, then, if you remove the user and then click create, the user reappears briefly. They don't get added to the project, but it's confusing.

All the other values are read in this `onMount` too, just to make sure that any changes the admin make don't magically get overwritten by reactive updates.